common/v2/features/DeFiZap/components/ZapIndicators.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { SPACING, COLORS } from 'v2/theme';
+
+import bullishIndicator from 'assets/images/defizap/indicators/bullishIndicator.svg';
+import bearishIndicator from 'assets/images/defizap/indicators/bearishIndicator.svg';
+
+const IContainer = styled.div`
+  display: flex;
+`;
+
+const IImage = styled.img`
```

Comment:
We usually go with `SImage` or simply `Image` if that makes more sense.

Nitpicky, sorry, but good to have a general consensus on this.